user_docs/en/userGuide.t2t

```diff
@@ -1296,16 +1296,19 @@ Some settings can also be changed using shortcut keys, which are listed where re
 
 ++ NVDA Settings ++[NVDASettings]
 %kc:settingsSection: || Name | Desktop key | Laptop key | Description |
-The NVDA Settings dialog contains many configuration parameters that can be changed.
-This dialog contains a list with several categories of settings to choose from.
-When you select a category, several settings related to this category will be shown in this dialog.
-These settings can be applied using the apply button, in which case the dialog will stay open.
+NVDA provides many configuration parameters that can be changed using the settings dialog.
+To make it easier to find the kind of settings you want to change, the dialog displays a list with several configuration categories to choose from.

```

Comment:
```suggestion
To make it easier to find the kind of settings you want to change, the dialog displays a list of configuration categories to choose from.
```

The list contains 15 items now, not counting any extras added by add-ons.  That seems like more than "several" categories